src/classes/atomTypeMix.cpp

```diff
@@ -56,7 +57,7 @@ void AtomTypeMix::add(const AtomTypeMix &source)
     // Loop over AtomTypes in the source list
     for (auto &otherType : source)
     {
-        AtomTypeData &atd = add(otherType.atomType());
+        auto &&[atd, atdIndex] = add(otherType.atomType());
```

Comment:
```suggestion
        auto &&[atd, std::ignore] = add(otherType.atomType());
```
Since we never use the `atdIndex`, it's best just to ignore it.

Reply:
:+1:  Had no idea this was a thing!

Reply:
@rprospero ...although annoyingly you cannot use `std::ignore` in a structured binding (yet). As a middle ground I'll refactor these to be `auto &atd = std::get<0>(...)`